fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -161,9 +202,60 @@ class BrowserToolbarCFRPresenter(
             },
         ).run {
             settings.shouldShowTotalCookieProtectionCFR = false
-            tcpCfrPopup = this
+            popup = this
             show()
             TrackingProtection.tcpCfrShown.record(NoExtras())
         }
     }
+
+    @VisibleForTesting
+    internal fun showShoppingCFR(shouldShowOptedInCFR: Boolean) {
+        CFRPopup(
+            anchor = toolbar.findViewById(
+                R.id.mozac_browser_toolbar_page_actions,
+            ),
+            properties = CFRPopupProperties(
+                popupWidth = 475.dp,
```

Comment:
what happens with this width parameter on devices smaller than 475? will it properly stop at the device edges?

Reply:
Because `PopupAlignment.BODY_CENTERED_IN_SCREEN` is set, 475 is used as maximum width. For smaller screens, the CFR expands up to `16.dp` from the screen edges.